src/routes/FruitDetails.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { useLoaderData } from 'react-router-dom';
+import style from '../style/FruitDetails.module.css';
+import Header from '../components/Header';
+
+export async function loader({ params }) {
+  return fetch(
+    `https://api.allorigins.win/raw?url=https://www.fruityvice.com/api/fruit/${params.id}`
+  );
+}
+
+export default function FruitDetails() {
+  const fruitData = useLoaderData();
+  const fruit = JSON.parse(fruitData);
+
+  console.log(fruit);
+  return (
+    <>
+      <Header />
+      <div className={style.container}>
+        <div className={style.main}>
+          <h1>{fruit.name}</h1>
+          <div>
+            <strong>Family:</strong>{' '}
+            {fruit.family != 0 ? fruit.family : 'No information available'}
+          </div>
+          <div>
+            <strong>Genus:</strong>{' '}
+            {fruit.genus != 0 ? fruit.genus : 'No information available'}
+          </div>
+          <div>
+            <strong>Order:</strong>{' '}
+            {fruit.order != 0 ? fruit.order : 'No information available'}
+          </div>
+        </div>
+        {fruit.nutritions.calories ? (
+          <div className={style.nutrition}>
+            <h2>Nutrition Information</h2>
+            <div>
+              <strong>Calories:</strong> {fruit.nutritions.calories}
+            </div>
+            <div>
+              <strong>Carbohydrates:</strong> {fruit.nutritions.carbohydrates}
+            </div>
+            <div>
+              <strong>Fat:</strong> {fruit.nutritions.fat}
+            </div>
+            <div>
+              <strong>Protein:</strong> {fruit.nutritions.protein}
+            </div>
+            <div>
+              <strong>Sugar:</strong> {fruit.nutritions.sugar}
+            </div>
+          </div>
```

Comment:
- [ ] [REQUIRED] Kindly note that the project requirement states that you should select one main color for the app and be consistent. Kindly remove this color #bde0fe and replace it with a shade of the major color you chose just like you did on the `Home` page.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-06 23-15-10](https://user-images.githubusercontent.com/58692590/194429712-f935895a-fe2d-47f2-9fea-5a88b2ba1d7f.png) | ![Screenshot from 2022-10-06 23-14-51](https://user-images.githubusercontent.com/58692590/194429866-eb3e3bb1-0f0d-4fa7-bd25-a7e657f0253b.png)